### PR TITLE
Remove Rinkeby from our config completely

### PR DESCRIPTION
**Pull request by Devorein**

### CODE REVIEW

File: components/[pageId]/DocumentPage/components/BountyProperties/components/BountyApplicantsTable/BountyPaymentButton.tsx [hunk 1]
```diff
@@ -234,23 +235,30 @@ export default function BountyPaymentButton ({
     }
   };
 
+  const isChainSupported = !unsupportedChainIds.includes(bounty.chainId);
```
Comment (mattcasey) [inline components/[pageId]/DocumentPage/components/BountyProperties/components/BountyApplicantsTable/BountyPaymentButton.tsx:238]: After this PR, by definition we should not longer need this config 'unsuportedChainIds'. There's lots of other chains we don't support that are already handled (for example i just added Optimism last week)

File: scripts/transferChain.ts [hunk 2]
```diff
@@ -0,0 +1,24 @@
...
+        in: unsupportedChainIds
+      }
+    },
+    // Transfer to goerli
+    data: {
+      chainId: 5
+    }
+  });
+
+  await prisma.paymentMethod.deleteMany({
```
Comment (mattcasey) [inline scripts/transferChain.ts:17]: just want to confirm: we don't link to payment methods right? it's just we let you use them to select a chainId for bounties, but the bounty update above will address that

File: components/[pageId]/DocumentPage/components/BountyProperties/components/BountyApplicantsTable/BountyPaymentButton.tsx (hunk 1 above)
Comment (Devorein) [reply components/[pageId]/DocumentPage/components/BountyProperties/components/BountyApplicantsTable/BountyPaymentButton.tsx:238]: Got it. I will remove those.

File: scripts/transferChain.ts (hunk 2 above)
Comment (Devorein) [reply scripts/transferChain.ts:17]: Exactly, we only use payment method for selecting the chainId. Nothing is linked according to the schema

**Pull request merged**